features/step_definitions/metering.rb

```diff
@@ -125,3 +125,40 @@
     raise "report '#{report_type}' for project '#{namespace}' not found after #{seconds} seconds"
   end
 end
+
+Given /^I enable route for#{OPT_QUOTED} metering service$/ do | metering_name |
+  metering_name ||= "openshift-metering"
+  htpasswd = `htpasswd -nb -s #{user.name} #{user.password}`.strip
```

Comment:
this will fail for token users (although I discourage such users usage anyway). Maybe use just some random user/pass and put into clipboard?

Reply:
now I see this is running local command which is not nice, need proper string generated by ruby

Reply:
See format of passwd file [1]. Perhaps we can use OpenSSL gem to generate one of these. I can check this next week if you don't figure it out.

[1] https://httpd.apache.org/docs/trunk/misc/password_encryptions.html

Reply:
This is how OpenSSh hashing can be used:
https://docs.ruby-lang.org/en/2.3.0/OpenSSL/HMAC.html

Reply:
wow, now I notice there is an example how to create a SHA1 password in ruby in the apache docs file I wrote above, so it should be trivial to do

Reply:
Do you prefer the openssl approach or what's in the apache doc?

Reply:
@akostadinov, perhaps add this to `rand_str` method, although that would require adding an optional parameter for passing in the `password`, WDYT?

Reply:
on 2nd thought, that probably does not belong in that method.... perhaps a new one call encrypt?